src/moin/utils/mimetype.py

```diff
@@ -79,6 +79,9 @@
     ("application", "x-tex"): ("text", "tex"),
     ("application", "javascript"): ("text", "javascript"),
     ("application", "x-dos-batch"): ("text", "bat"),
+    # Non-standard MIME types used on some platforms
+    ("application", "x-gzip"): ("application", "gzip"),  # Windows, macOS
+    ("application", "x-zip-compressed"): ("application", "zip"),  # Windows
```

Comment:
I'm not sure that is correct.

x-zip is normally abbreviated as xz and is not equal to zip